src/sql/workbench/browser/modal/calloutDialog.ts

```diff
@@ -103,224 +52,32 @@ export class CalloutDialog extends Modal {
 				dialogProperties: dialogProperties,
 				width: width
 			});
-
-		this._selectionComplete = new Deferred<ICalloutDialogOptions>();
-		this._calloutType = calloutType;
 	}
 
 	/**
 	 * Opens the dialog and returns a promise for what options the user chooses.
```

Comment:
This description doesn't really make sense anymore.

And given that this is only calling show I'd honestly just remove it completely. The superclasses can have their own open functions if they want but it doesn't seem necessary for the base class here